azure-fs-aks/Program.fs

```diff
@@ -0,0 +1,131 @@
+﻿module Program
+
+open Pulumi.FSharp
+open Pulumi.Azure.Core
+open Pulumi.Azure.ContainerService
+open Pulumi.Azure.ContainerService.Inputs
+open Pulumi.Azure.AD
+open Pulumi.Azure.Network
+open Pulumi.Random
+open Pulumi.Tls
+
+[<RequireQualifiedAccess>]
+module Pulumi =
+    let createResourceGroup name = ResourceGroup(name)
+
+    let createPassword() =
```

Comment:
Consider accepting `name` as an argument

Reply:
I was trying to keep the helpers simple, but I agree, I should be consistent and at least have name on all helpers.